### ISSUE TITLE
(Suggestion) Have Non-SRD items coded using DAE and midi-qol

### ISSUE THREAD

**Issue by nowfocus** [MrPrimate/ddb-importer#124]
Lots of items that are imported could easily have the appropriate effects added.  My example today was Warcaster - non-SRD content, simple to implement using midi-qol, but cannot be included in other modules because they cannot verify purchase.  

Seems like a great feature to have here where purchase can be verified.  The community could help suggest how to code the effects properly.

**Comment by MrPrimate** [comment]
@nowfocus yep, please reach out on Discord if you have any of these you wish to share.
